### ISSUE TITLE
Failed to update ui::AXTree in accessiblility_bridge.cc(76)

### ISSUE THREAD

**Issue by timsneath** [flutter/flutter#98778]
## Steps to Reproduce
Running a relatively simple Windows app example, I'm getting repeated accessibility bridge errors. They aren't fatal to the app (but maybe to accessibility itself, I don't know). 

App example is here: 
https://github.com/timsneath/win32_gamepad/tree/b4c96b699e562424203117515eb2ea3991da9bba/example/inspector

## Error
```
Restarted application in 373ms.
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 29 is not marked for destruction, would be reparented to 3
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: 18 will not be in the tree and is not the new root
```

## Flutter doctor results
```
[√] Flutter (Channel stable, 2.10.1, on Microsoft Windows [Version 10.0.22557.1], locale en-US)
    • Flutter version 2.10.1 at c:\flutter
    • Upstream repository git@github.com:flutter/flutter.git
    • Framework revision db747aa133 (9 days ago), 2022-02-09 13:57:35 -0600
    • Engine revision ab46186b24
    • Dart version 2.16.1
    • DevTools version 2.9.2

[√] Chrome - develop for the web
    • Chrome at C:\Program Files (x86)\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Community 2022 17.0.5)
    • Visual Studio at C:\Program Files\Microsoft Visual Studio\2022\Community
    • Visual Studio Community 2022 version 17.0.32112.339
    • Windows 10 SDK version 10.0.22000.0

[√] VS Code (version 1.64.2)
    • VS Code at C:\Users\tim\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.34.0

[√] Connected device (3 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.22557.1]
    • Chrome (web)      • chrome  • web-javascript • Google Chrome 98.0.4758.102
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 98.0.1108.50

[√] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```

**Comment by gspencergoog** [comment]
cc @cbracken

**Comment by maheshmnj** [comment]
This looks like a duplicate of https://github.com/flutter/flutter/issues/98099

**Comment by rydmike** [comment]
@maheshmnj and @timsneath, yes I think it is the same issue too.

**Comment by maheshmnj** [comment]
Closing this issue as a duplicate of #98099. In case you disagree please feel free to reopen.
